task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   annotations:
     tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: "konflux"
+    build.appstudio.redhat.com/expires-on: "2025-01-31T00:00:00Z"
```

Comment:
Will it be possible to announce this also on users channel for general awareness?